src/views/clusteroverview/overview/components/details-card/utils/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const OPENSHIFT_VIRT_PATH =
```

Comment:
this is the wrong link, it will link to the list, you need to link to the installed opertor.
for example:
if the operator installed:
has - 
name: `kubevirt-hyperconverged-operator.v4.11.0`
namespace: `openshift-cnv`
the link should be -
`/k8s/ns/openshift-cnv/operators.coreos.com~v1alpha1~ClusterServiceVersion/kubevirt-hyperconverged-operator.v4.11.0`

Reply:
Fixed! (I was going by what the BZ said.)